docs/infrastructure/deployment-targets/amazon-ecs-cluster-target.md

```diff
@@ -6,10 +6,24 @@ position: 30
 
 ECS Cluster targets are used by the [ECS steps](/docs/deployments/aws/index.md) to define the context in which deployments and scripts are run.
 
+:::hint
```

Comment:
I'm thinking might actually need to split this out into a separate PR as we are planning on shipping changes for Azure target discovery before ECS.